sdcm/sct_config.py

```diff
@@ -296,6 +296,9 @@ class SCTConfiguration(dict):
         dict(name="use_ms_ad_ldap", env="SCT_USE_MS_AD_LDAP", type=boolean,
              help="This option will use ldap server of QA MS Active Directory, not default openldap"),
 
+        dict(name="use_mgmt_meta_installation", env="SCT_USE_MGMT_META_INSTALLATION", type=boolean,
```

Comment:
shouldn't that be the default now, and have the "legacy" code for previous upgrades only, until the oldest supported version will be 2.6 (that will be when 2.8 is released)?

Reply:
You seem to have those two mixed up. The meta installation is the method that will no longer be supported from now on. Only the way we have been installing until now will be supported. So we don't have to change anything further.

Reply:
so seems something is missing in explaining this feature, and why it is being introduced to 2.6 and will not be supported anymore in the future...

Reply:
It always existed. We just never used it.
It was not introduced in 2.6, on the contrary. Meta installation is being removed in 2.6. Which is why I added this test - to see that when a manager sever that was installed with meta installation is being upgraded to 2.6, nothing goes wrong.

`//TODO: Should be deprecated once 2.6 is no longer tested, since 2.6 doesn't even support meta installation anymore.`

Reply:
please, add this `TODO` or `FIXME` to here